src/open_petro_elastic/__main__.py

```diff
@@ -335,6 +335,11 @@ def parse_arguments(argv):
         + " Unsuccessful rows will be given NaN. Note that "
         + "this option can deteriorate performance.",
     )
+    return ap
+
+
+def parse_arguments(argv):
+    ap = make_parser(argv[0])
```

Comment:
Slightly outside the scope, but you dont have to call these with argv

Reply:
True, but the `argv` name gives it the right intention, and its practical to expose the possibility of replacing `sys.argv`.